mtools/mplotqueries/mplotqueries.py

```diff
@@ -81,7 +84,7 @@ def __init__(self):
                                     help=("specify value to group on. "
                                           "Possible values depend on type of "
                                           "plot. All basic plot types can "
-                                          "group on 'namespace', 'operation', "
+                                          "group on 'namespace','connector' 'operation', "
```

Comment:
What do you mean by _connector_ here?

Reply:
Agreed. It's meant to be hostname. Improper terminology there from me. I'll update that to hostname

Reply:
Is there a missing comma before operations? My understanding id that you can name by `namespace`, `hostname`, `operation`